Sources/Interpreter/Interpreter.swift

```diff
@@ -276,9 +304,10 @@ public struct Interpreter {
       popStackFrame()
       return
     case let x as Store:
-      _ = x
+      try memory.store(asBuiltinValue(x.object)!, at: asAddress(x.target)!, with: &typeLayouts)
```

Comment:
This would read better, IMO.  Not worth doing unless we find the pattern repeating a lot though:

```suggestion
      try memory.store(
        x.object.asBuiltinValue(in: self), at: x.target.asAddress(in: self), 
        with: &typeLayouts)
```